fabric-textures-v0/src/main/java/net/fabricmc/fabric/api/client/texture/ColorSniffer.java

```diff
@@ -0,0 +1,238 @@
+/*
+ * Copyright (c) 2016, 2017, 2018, 2019 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.fabric.api.client.texture;
+
+import net.fabricmc.fabric.mixin.client.texture.SpriteAccessor;
+import net.minecraft.client.texture.NativeImage;
+import net.minecraft.client.texture.Sprite;
+
+import java.awt.*;
```

Comment:
Ah yes, java.awt. macOS support is no more

Reply:
macos has AWT pretty sure

Reply:
Using AWT classes within Minecraft on macOS seems to break parts of LWJGL